src/utils/coordinates/coordinateExtractors.js

```diff
@@ -40,20 +39,20 @@ const webmercatorExtractor = (regexMatches) => {
     }
     let lon, lat
     if (regexMatches.length === 5) {
-        // 4 matches + global match, i.e. : (47)°(5.123)', (8)°(4.154)' (we inverse lat/lon to lon/lat in the process)
-        lon = Number(regexMatches[3]) + Number(regexMatches[4]) / 60.0
-        lat = Number(regexMatches[1]) + Number(regexMatches[2]) / 60.0
+        // 4 matches + global match, i.e. : (47)°(5.123)', (8)°(4.154)'
+        lon = Number(regexMatches[1]) + Number(regexMatches[2]) / 60.0
+        lat = Number(regexMatches[3]) + Number(regexMatches[4]) / 60.0
     }
     if (regexMatches.length === 7) {
-        // 6 matches + global match, i.e. : (47)°(5)'(41.61)", (8)°(4)'(6.32)" (we inverse lat/lon to lon/lat in the process)
+        // 6 matches + global match, i.e. : (47)°(5)'(41.61)", (8)°(4)'(6.32)"
```

Comment:
Why removing the inverse of lat/lon here ? Would be good to document it in the commit message 😉

Reply:
As we now not know the target projection (or default projection) I thought it would make sense to always return coordinate as `[x, y]` (meaning `[lon, lat]`) instead of doing some specifics for WGS84 and return `[lat, lon]` because that's how some of our frameworks want that. I prefer that we invert this coordinates when they need to be (for OpenLayers for instance) but store them as X, Y, much more logical